protocol/java/org/openyolo/protocol/HintRetrieveResult.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright 2017 The OpenYOLO Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openyolo.protocol;
+
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.openyolo.protocol.Protobufs.HintRetrieveResult.ResultCode;
+import org.openyolo.protocol.internal.AdditionalPropertiesUtil;
+import org.openyolo.protocol.internal.ByteStringConverters;
+import org.openyolo.protocol.internal.CollectionConverter;
+
+/**
+ * A response to a {@link HintRetrieveRequest hint request}. The status code indicates the outcome
+ * of the request, and the response may contain a hint.
+ */
+public final class HintRetrieveResult {
+
+    /**
+     * Indicates that the provider returned a response that could not be interpreted.
+     */
+    public static final int CODE_UNKNOWN = ResultCode.UNSPECIFIED_VALUE;
```

Comment:
Replace "CODE" with "RESULT" to match previously established pattern (https://github.com/openid/OpenYOLO-Android/commit/2c5082989ca2c336e2209b1426519b780269c112)